test/users.js

```diff
@@ -0,0 +1,195 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  sessionManager = require('../app/services/sessionManager'),
+  should = chai.should();
+
+const successfulLogin = callback => {
```

Comment:
Can't you reuse this function? it seems the same than the game's login function

Reply:
now i added a new script js called utils.js with the login function